### PR TITLE
Domain types

### PR DESCRIPTION
**Pull request by mateuszmidor**
KNTK-221
Based on https://github.com/kentik/sla_dashboard_webapp/pull/8

### CODE REVIEW

File: domain/model/mesh_results.py
Comment (Fenthick) [inline domain/model/mesh_results.py:19]: I think it is better to name it "agent_name" instead of adding comment, since to comment is visible only in one place. The same with agent alias and fields in MeshRow class.

**Review by Fenthick — COMMENTED** [review]
Nitpick.

File: domain/model/mesh_results.py
Comment (mmac-m3a) [reply domain/model/mesh_results.py:19]: Note that the `name` field returned in `Agent{List,Get}` data is actually internal and should not be shown externally. It will likely be removed soon. There is a ticket open with engineering to include more metadata for agents in the API, so that they can be more uniquely identified with human consumable strings. I propose to use combination of `alias`, `id` and `asn` as "name" presented to users for now, like for example so:
```
alias (id: <id> asn: <asn>)
```

**Review by mmac-m3a — APPROVED** [review]
Fine with me. I though still hope my comments in #8 will get addressed soon(ish).

**Comment by mmac-m3a** [comment]
Waiting for approval from Codilime team before merging.

File: infrastructure/data_access/http/synthetics_repo.py
Comment (mateuszmidor) [reply infrastructure/data_access/http/synthetics_repo.py:29]: updating

File: domain/model/mesh_results.py
Comment (Fenthick) [reply domain/model/mesh_results.py]: Done

**Review by Fenthick — APPROVED** [review]

**Review by lukaa12 — APPROVED** [review]

**Review by Fenthick — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**
